recipe/activate.sh

```diff
@@ -0,0 +1,3 @@
+# CppUTest will look for this variable to find the library
+# https://cpputest.github.io/manual.html
+export CPPUTEST_HOME="$CONDA_PREFIX"
```

Comment:
```suggestion
export CPPUTEST_HOME="$CONDA_PREFIX"

```

Newline at the end of file :)